core/services/gateway/handlers/functions/orm.go

```diff
@@ -130,3 +134,37 @@ func (cs *cachedSubscriptionRow) encode() CachedSubscription {
 		},
 	}
 }
+
+func (o *orm) GetAllowedSenders(offset, limit uint, qopts ...pg.QOpt) ([]common.Address, error) {
+	var address []common.Address
+	stmt := fmt.Sprintf(`
+		SELECT allowed_address
+		FROM %s
+		WHERE router_contract_address = $1
+		ORDER BY id ASC
+		OFFSET $2
+		LIMIT $3;
+	`, allowlistTableName)
+	err := o.q.WithOpts(qopts...).Select(&address, stmt, o.routerContractAddress, offset, limit)
+	if err != nil {
+		return address, err
+	}
+
+	return address, nil
+}
+
+func (o *orm) UpsertAllowedSender(id uint64, allowedSender common.Address, qopts ...pg.QOpt) error {
```

Comment:
yes, does not make sense anymore, thanks for point it it out.

I've also modify the name of the method to `CreateAllowedSender` because we won't be updating an allowed sender